dir.common.props

```diff
@@ -98,4 +78,25 @@
 
   </PropertyGroup>
 
+  <!-- Version numbers for binaries and packages
+       [ARCADE REMOVE] These properties can be removed once native binaries use Arcade versioning
+       They are included in common for packaging, which is onboarded to Arcade, but uses the version
+       numbers for native binaries for some outputs. This should be kept in sync or moved to eng/Versions.props -->
+  <PropertyGroup>
+    <PackageVersion Condition="'$(PackageVersion)' == ''">3.0.0</PackageVersion>
+    <ProductVersion>$(PackageVersion)</ProductVersion>
+    <PreReleaseLabel>preview6</PreReleaseLabel>
+    <PreReleaseVersionLabel>$(PreReleaseLabel)</PreReleaseVersionLabel>
+    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
+    <StableVersion Condition="'$(StabilizePackageVersion)' == 'true' and '$(StableVersion)' == ''">$(PackageVersion)</StableVersion>
+    <NativeMajorVersion>4</NativeMajorVersion>
+    <NativeMinorVersion>30</NativeMinorVersion>
```

Comment:
Where did the "30" come from?

Reply:
@elinor-fung conversationally mentioned there were some there were plans to move the versioning for all files to 4.30, so I was trying the impact of it in our mixed build. That being said, this change shouldn't have those.

Reply:
@RussKeldorph had indicated that `4.30.<build>` was the plan when I had asked about what the file versions should be

Reply:
Sorry, I had outdated information.  We should just follow what CoreFX does for fileversion.  Specify 4.7, and arcade will multiply it to 4.700.  The one place CoreCLR shouldn't follow CoreFX is packages version.  CoreCLR should continue to use the branding version (3.0...) for that.